src/Windowing/OpenToolkit.Windowing.Common/Interfaces/IGameWindow.cs

```diff
@@ -15,7 +15,7 @@ namespace OpenToolkit.Windowing.Common
     /// <summary>
     /// Defines the interface for a GameWindow.
```

Comment:
This is the most hilariously bad doc comment ever.

Can we briefly explain how IGameWindow differs from INativeWindow?

"IGameWindow provides a game update loop and corresponding events on top of INativeWindow"

Reply:
I'd prefer doing that in another PR, since I'll probably touch the exact separation between Game and NativeWindow anyways, it'd be better to do it then.